website/demos/ScrollToCell.tsx

```diff
@@ -0,0 +1,82 @@
+import { useRef, useState } from 'react';
+
+import DataGrid from '../../src';
+import type { Column, FormatterProps, DataGridHandle } from '../../src';
+import type { Props } from './types';
+
+type Row = number;
+const rows: readonly Row[] = [...Array(200).keys()];
```

Comment:
Optional, but this an optimized way to do this (probably)
```suggestion
const rows: readonly Row[] = Array.from({ length: 200 }, (_, i) => i);
```